conda/base/constants.py

```diff
@@ -251,12 +251,16 @@ class SatSolverChoice(ValueEnum):
     PYSAT = 'pysat'
 
 
-class ExperimentalSolverChoice(ValueEnum):
+class SolverChoice(ValueEnum):
     CLASSIC = 'classic'
     LIBMAMBA = 'libmamba'
     LIBMAMBA_DRAFT = 'libmamba-draft'
 
 
+# TODO: Remove in a later release - compatibility alias
```

Comment:
Do we want to call out the planned release we want to remove this in?

Reply:
I'd like to do that, but I think we need to come up with a mechanism / protocol to handle this across the codebase. Either a specific comment syntax or a time-controlled unit test that keeps track and notifies us?

Reply:
I think it's a good idea that we automate this (perhaps a custom PyLint parser?), but this work is going to be very involved and should be handled separately.

I will create a spike issue for implementing a deprecation mechanism that can be detected by our CI workflow.